packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
+    justify-content: center;
+  }
+
+  &.hasMarginTop {
+    margin-bottom: 0;
+    margin-top: var(--spacing-xsmall);
+  }
+
+  &.is-mobile {
+    font: var(--text-body-small);
+    font-weight: 300;
+    letter-spacing: var(--text-body-small-letter);
+  }
+
+  * {
+    margin: 0;
+  }
+}
+
+.title {
+  font: var(--text-body-medium);
+  font-weight: 300;
+  letter-spacing: var(--text-body-medium-letter);
+
+  &.is-mobile {
+    font: var(--text-body-small);
+    font-weight: 300;
```

Comment:
block: we don't need it too

Reply:
https://github.com/juntossomosmais/atomium/pull/695#discussion_r1943656350